app/views/navbar/_navbar.html.erb

```diff
@@ -0,0 +1,11 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <div class="container">
+      <a class="navbar-brand" href="/">Brighcoders Events</a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+        <a class="btn btn-primary ms-lg-3" href="/events">Start</a>
```

Comment:
The button text should be more descriptive and meaningful. Instead of 'Start', consider using 'View Events' or something similar.